integration-test/src/test/java/org/uniprot/store/indexer/search/uniparc/UniParcSearchEngine.java

```diff
@@ -55,7 +55,7 @@ protected String identifierField() {
 
     @Override
     protected String identifierQuery(String entryId) {
-        return getSearchFieldConfig().getSearchFieldItemByName("upid").getFieldName()
+        return getSearchFieldConfig().getSearchFieldItemByName("upi").getFieldName()
                 + ":"
                 + entryId;
```

Comment:
I am bit confused here. Was this code buggy? Earlier we were calling getSearchFieldItemByName("upid").
If it was not buggy then we should call getSearchFieldItemByName("proteome") since upid is renamed to proteome?

Reply:
I think it was a buggy, because the identifier field for UniParc is upi. If you see the method above identifierField, it also uses upi. In this case, upi makes more sense than upid.